### PR TITLE
Fixes Spears Disconnecting client when droppped

### PR DESCRIPTION
**Pull request by Bod9001**
fix a few bugs with it not being networked properly too

CL: [Fix] fixed spears disconnecting client when dropped.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Items/Wieldable.cs [hunk 1]
```diff
@@ -75,20 +75,27 @@ public void OnInventoryMoveServer(InventoryMove info)
...
+				isWielded = false;
+				itemAttributes.ServerHitDamage = damageUnwielded;
+				itemAttributes.SetSprites(Unwielded);
+				HideHand((int) hiddenHandValues.none, info.FromPlayer.PlayerScript);
 			}
 		}
 
 		[Server]
-		private void HideHand(int HiddenHandSelection)
+		private void HideHand(int HiddenHandSelection, PlayerScript PlayerScript)
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/Wieldable.cs]:
```suggestion
		private void HideHand(int HiddenHandSelection, PlayerScript playerScript)
```

**Comment by corp-0** [comment]
Since you're there, can you make the methods take the enum value directly so we don't have to cast to int unnecessarily?

File: UnityProject/Assets/Scripts/Items/Wieldable.cs [hunk 2]
```diff
@@ -75,20 +75,27 @@ public void OnInventoryMoveServer(InventoryMove info)
...
+				isWielded = false;
+				itemAttributes.ServerHitDamage = damageUnwielded;
+				itemAttributes.SetSprites(Unwielded);
+				HideHand( HiddenHandValue.none, info.FromPlayer.PlayerScript);
 			}
 		}
 
 		[Server]
-		private void HideHand(int HiddenHandSelection)
+		private void HideHand(HiddenHandValue HiddenHandSelection, PlayerScript playerScript)
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/Wieldable.cs]:
```suggestion
		private void HideHand(HiddenHandValue hiddenHandSelection, PlayerScript playerScript)
```

File: UnityProject/Assets/Scripts/Items/Wieldable.cs [hunk 3]
```diff
@@ -75,20 +75,27 @@ public void OnInventoryMoveServer(InventoryMove info)
...
 		{
-			PlayerManager.LocalPlayerScript.PlayerOnlySyncValues.ServerSetHiddenHands(HiddenHandSelection);
+			playerScript.PlayerOnlySyncValues.ServerSetHiddenHands(HiddenHandSelection);
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/Wieldable.cs]:
```suggestion
			playerScript.PlayerOnlySyncValues.ServerSetHiddenHands(hiddenHandSelection);
```

**Review by corp-0 — APPROVED** [review]

**Review by NoooneyDude — APPROVED** [review]

**Pull request merged**